docs/docs/targeting.md

```diff
@@ -0,0 +1,196 @@
+Part of what makes Pathfora powerful is its ability to interact with [Lytics](http://www.getlytics.com/) and target certain audiences with certain modules. Setting up the targeting in Pathfora requires an object with certain targeting rules as the first param to [initializeWidgets](/api/methods.md#initializewidgets) and your Lytics Account ID as a second parameter.
+
+``` javascript
+var modules = {
+  target: [{
+    segment: 'smt_name',
+    widgets: [ module ]
+  }]
+};
+
+pathfora.initializeWidgets(modules, 'YOUR LYTICS ACCOUNT ID');
+```
+
+## Setup Audiences
+
+You'll need to have at least one audience built in Lytics that you want to target with a module. Make sure that you have API access enabled for the audience, and have entered an ID, you will use this id in the targeting rules.
```

Comment:
we should link to "enabling api access on a segment" docs (not sure if we have them or not yet)